kernel/build/build_rpm

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 set -ex
 
-# Only do actual work when we are an RPM distro
```

Comment:
@alfredodeza Oh, should have been more clear - because we already have validate_{deb,rpm} in the definition.